doc/essentials/EnergyPlusEssentials.tex

```diff
@@ -0,0 +1,2260 @@
+
+\subsubsection*{Graphical User Interface}
+
+When running an EnergyPlus simulation within a graphical user interface,
+the exact method will vary depending on the specific program being
+used. You should read the documentation for that software to understand
+how to perform a simulation. 
+
+\subsubsection*{Command Line}
+
+EnergyPlus can be used as a command line tool within a Terminal window
+in Linux or MacOS or with the CMD prompt or PowerShell window under
+Windows. Basic usage using the command line approach is well documented
+in the QuickStart Guide. To learn more about the command line mode,
+you can type:
+\begin{verbatim}
+energyplus --help 
+\end{verbatim}
+when in the Energyplus folder. This will give the following display
+of options:
+\begin{verbatim}
+EnergyPlus, Version 9.0.1-bb7ca4f0da
+Usage: energyplus [options] [input-file]
+Options:
+  -a, --annual                 Force annual simulation
+  -c, --convert                Output IDF->epJSON or epJSON->IDF, dependent on
+                               input file type
+  -d, --output-directory ARG   Output directory path (default: current
+                               directory)
+  -D, --design-day             Force design-day-only simulation
+  -h, --help                   Display help information
+  -i, --idd ARG                Input data dictionary path (default: Energy+.idd
+                               in executable directory)
+  -m, --epmacro                Run EPMacro prior to simulation
+  -p, --output-prefix ARG      Prefix for output file names (default: eplus)
+  -r, --readvars               Run ReadVarsESO after simulation
+  -s, --output-suffix ARG      Suffix style for output file names (default: L)
+                                  L: Legacy (e.g., eplustbl.csv)
+                                  C: Capital (e.g., eplusTable.csv)
+                                  D: Dash (e.g., eplus-table.csv)
+  -v, --version                Display version information
+  -w, --weather ARG            Weather file path (default: in.epw in current
+                               directory)
+  -x, --expandobjects          Run ExpandObjects prior to simulation
+Example: energyplus -w weather.epw -r input.idf
+\end{verbatim}
+EnergyPlus can be run by specifying a number of options followed by
+the path to the input file. The file itself is usually in IDF (Input
+Data File) format or epJSON format, but it may also be in IMF (Input
+Macro File) format to be run with EPMacro using the -{}-epmacro option.
+Each option has a short form (a single-character preceded by a single
+dash, e.g., \textquotedbl -h\textquotedbl ) and a long form (a more
+descriptive string of characters preceded by double dashes, e.g.,
+\textquotedbl -{}-help\textquotedbl ). Several of these options
+are commonly used including the weather, output-prefix, expandobjects,
+and readvars options. The following are some examples of using the
+command line options. 
+
+Using a custom IDD file:
+\begin{verbatim*}
```

Comment:
These are rendering a bit funny, with spaces being printed as [open spaces](https://en.wiktionary.org/wiki/%E2%90%A3).  Again, this may be addressed once this document brings in the standard E+ template.

![image](https://user-images.githubusercontent.com/849824/52083389-e7f41080-2564-11e9-9339-d6b61d9f6486.png)

Reply:
Changed verbatim* to verbatim and that seems to get rid of the space indicators